std-bits/table/src/main/java/org/enso/table/aggregations/Aggregator.java

```diff
@@ -7,9 +7,7 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-/***
- * Interface used to define aggregate columns.
- */
+/** * Interface used to define aggregate columns. */
```

Comment:
Can you clean this file up a bit? A lot of comments were using triple-star syntax.